common/culture/innovations/wc_cultural_maa_innovations.txt

```diff
@@ -45,3 +47,863 @@ innovation_taming_the_skies = {
+	culture_era = culture_era_tribal
+	icon = @special_maa_01
+
+	potential = {
+		this.culture_group = culture_group:orc_group
+	}
+
+	unlock_maa = wyvern_rider
+
+	flag = global_maa
+}
+
+
+####Early Medival
+
+innovation_baradian_marine = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		OR = {
+			this = culture:tirassian
+			this = culture:baradin
+		}
+	}
+
+	unlock_maa = baradian_marine
+
+	flag = global_maa
+}
+
+
+innovation_blackrock_legionnaire = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		this = culture:blackrock
+	}
+
+	unlock_maa = blackrock_legionnaire
+
+	flag = global_maa
+}
+
+innovation_chaosborn = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		OR = {
+			this = culture:eredar
+			this = culture:eredruin
+		}
+	}
+
+	unlock_maa = chaosborn
+
+	flag = global_maa
+}
+
+innovation_dragonhawk = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		OR = {
+			this = culture:blood_elf
+			this = culture:high_elf
+		}
+	}
+
+	unlock_maa = dragonhawk_rider
+
+	flag = global_maa
+}
+
+innovation_farstriders = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		OR = {
+			this = culture:high_elf
+			this = culture:blood_elf
+		}
+	}
+
+	unlock_maa = farstriders
+
+	flag = global_maa
+}
+
+innovation_fel_guard = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		OR = {
+			this = culture:annihilan
+			this = culture:nathrezim
+			this = culture:shivarra
+			this = culture:observer
+			this = culture:sayaadi
+			this = culture:satyr
+		}
+	}
+
+	unlock_maa = fel_guard
+
+	flag = global_maa
+}
+
+innovation_forsaken_ranged = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		this = culture:forsaken
+	}
+
+	unlock_maa = dark_ranger
+
+	flag = global_maa
+}
+
+innovation_golem_arcane = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		this = culture:dalaranian
+	}
+
+	unlock_maa = golem_arcane
+
+	flag = global_maa
+}
+
+innovation_golem_mech = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		this.culture_group = culture_group:dwarf_group
+	}
+
+	unlock_maa = golem_mech
+
+	flag = global_maa
+}
+
+innovation_gryphon = {
+	group = culture_group_regional
+	culture_era = culture_era_early_medieval
+	icon = @special_maa_01
+
+	potential = {
+		OR = {
+			this.culture_group = culture_group:human_group
+			this = culture:wildhammer
+		}
+	}
+
+	unlock_maa = gryphon_rider
+
+	flag = global_maa
+}
```

Comment:
Here is a text from WC3 RoC manual on Gryphon Riders:
> The daring dwarves of the Wildhammer clan have responded once again to the call and brought the mighty gryphons of the Aerie Peaks to aid the Alliance in its time of need. Armed with their trusty, lightning-powered Stormhammers, the fearless wilddwarves seek to keep the skies of Lordaeron free from enemy forces. The proud gryphons share their riders' implacable resolve and stand as noble symbols of the Alliance's fortitude.

It wasn't until WoW that we get to see Bronzebeard or Ironforge Gryphons. This is acknowledged on the wiki:
> Gryphon riders are mostly dwarves of the Wildhammer clan, but more recently Ironforge dwarves and humans alike have started deploying gryphon riders of their own.
(https://wowpedia.fandom.com/wiki/Gryphon)

Reply:
That would still qualify them for the tech, I would just give it to them later in history.  Is there any dwarf that I should leave out of the tech?

Reply:
@TheGroxEmpire I know, I read the texts as well. So it mentions Gryphons for humans and Ironforge dwarves at roughly the same time, right? So they would be researchable for IF as well as for humans.
@Gaben1025 I can't think of dwarves not being able to research it. I don't know about the Darkiron dwarves, since they in lore don't use them, right? But I'd not ban them from using gryphons in general, to avoid too much railroading.

Reply:
@TheGroxEmpire @arithon Made all dwarves and humans unlock gryhpon at 583 and gave wildhammer gryphons at 1.  Sounds okay?

Reply:
Sounds good to me

Reply:
As I wrote in the new review, Humans and non-wildhammer dwarves should have access at the 3rd war, not earlier. So, 603.